nbconvert/preprocessors/regexremove.py

```diff
@@ -20,16 +20,14 @@ class RegexRemovePreprocessor(Preprocessor):
     of unicode strings. If the contents match any of the patterns, the cell
     is removed from the notebook.
 
-    By default, `patterns = [r'\Z']` which matches the empty string such that
-    strictly empty cells are removed. To modify the list of matched patterns,
+    To modify the list of matched patterns,
     modify the patterns traitlet. For example, execute the following command
-    to convert a notebook to html and remove cells containing only whitespace:
+    to convert a notebook to html and remove cells containing only whitespace::
 
-    > jupyter nbconvert --RegexRemovePreprocessor.enabled=True \
```

Comment:
The [default value](https://github.com/jupyter/nbconvert/blob/308fb2914a6528e44cffb9a697bac9d185e14fc3/nbconvert/preprocessors/base.py#L27) for the `enabled` flag appears to be `False`. We should either override the default value for the `RegexRemovePreprocessor` or keep the `--RegexRemovePreprocessor.enabled=True` flag. Or is a preprocessor enabled automatically if one of its attributes (`patterns`) is set?

Reply:
@tillahoffmann Thanks for the review!

I'm not quite sure, but I think the preprocessor is enabled by default because of this:

https://github.com/jupyter/nbconvert/blob/9f5c402b85cffdc3599a426fa13e9163946c6e2c/nbconvert/exporters/templateexporter.py#L126-L135

I don't care if it is enabled or not, as long as it doesn't remove stuff by default.